WinUIGallery/Controls/TileGallery.xaml

```diff
@@ -164,36 +178,31 @@
                         Description="Accelerates the creation of new WinUI apps using a wizard-based experience.."
                         Link="https://marketplace.visualstudio.com/items?itemName=TemplateStudio.TemplateStudioForWinUICs">
                         <local:HeaderTile.Source>
-                            <PathIcon Data="{StaticResource GitHubIconPath}" Foreground="{ThemeResource TextFillColorPrimaryBrush}">
+                            <PathIcon Data="{StaticResource TemplateStudioIconPath}" Foreground="{ThemeResource TextFillColorPrimaryBrush}">
                                 <PathIcon.RenderTransform>
-                                    <CompositeTransform
-                                        ScaleX="2.05"
-                                        ScaleY="2.05"
-                                        TranslateX="-20" />
+                                    <CompositeTransform ScaleX="1.2" ScaleY="1.2" />
                                 </PathIcon.RenderTransform>
                             </PathIcon>
                         </local:HeaderTile.Source>
                     </local:HeaderTile>
                     <local:HeaderTile
                         Title="Code samples"
                         Description="Find samples that demonstrate specific tasks, features, and APIs."
-                        Link=" https://partner.microsoft.com/en-us/dashboard/home">
+                        Link="https://docs.microsoft.com/windows/apps/get-started/samples">
                         <local:HeaderTile.Source>
                             <FontIcon
-                                FontSize="40"
+                                Margin="0,8,0,0"
+                                FontSize="44"
                                 Foreground="{ThemeResource TextFillColorPrimaryBrush}"
                                 Glyph="&#xE943;" />
                         </local:HeaderTile.Source>
                     </local:HeaderTile>
                     <local:HeaderTile
                         Title="Partner Center"
                         Description="Upload your app to the Store."
-                        Link="https://docs.microsoft.com/windows/apps/get-started/samples">
+                        Link=" https://partner.microsoft.com/en-us/dashboard/home">
```

Comment:
Maybe it would be better to point to the landing page or similar of the partner center? I don't think that it's a good experience if the first thing people are required to do when opening this link is signing in to their MSFT account. 

Also, trailing space.

Reply:
Hmm good point.. reverted this back to: https://developer.microsoft.com/windows/ which has a "Dashboard" link at the top.